docs/relations/sponsors.mdx

```diff
@@ -2,27 +2,48 @@
 sidebar_position: 0
 ---
 
+import ThemedImage from '@theme/ThemedImage'
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
 # Betaflight Sponsors
 
-<div className="grid xs:grid-cols-4 md:grid-cols-4 gap-4 mb-4 items-center justify-center h-full">
-  <a href="https://www.hqprop.com/" target="_blank" rel="noreferrer noopener" className="xl:col-span-1 col-span-2 no-effect">
-    <img src="/img/betaflight/sponsors/hqprop.svg" alt="HQProp" className="invert no-effect rounded-2xl w-4/5" />
-  </a>
-  <a href="https://https://www.radiomasterrc.com/" target="_blank" rel="noreferrer noopener" className="xl:col-span-1 col-span-2 no-effect">
-    <img src="/img/betaflight/sponsors/radiomaster-logo.jpg" alt="Radiomaster" className="no-effect rounded-2xl w-full" />
-  </a>
-  <a href="https://www.tititop.com/" target="_blank" rel="noreferrer noopener" className="xl:col-span-1 col-span-2 no-effect">
-    <img src="/img/betaflight/sponsors/dogcom.svg" alt="Radiomaster" className="no-effect rounded-2xl w-full" />
-  </a>
-  <a href="https://www.axisflying.com/" target="_blank" rel="noreferrer noopener" className="xl:col-span-1 col-span-2 no-effect">
-    <img src="/img/betaflight/sponsors/Axisflying_dark.svg" alt="AxisLogo" className="no-effect rounded-2xl w-full" />
+<div className="flex flex-col items-center backdrop-blur-md shadow-xl p-4 rounded-2xl bg-neutral-500/10 grid md:grid-cols-4 gap-4 mb-4">
```

Comment:
changing `flex-col` to `flex-cols-4` and removing `grid md:grid-cols-4` fixes the behaviour with very small screen widths.

Reply:
Please ignore the previous comment, must have been an image caching problem; sizing of this image now seems OK with code in the PR.